src/EA.Weee.RequestHandlers/Organisations/DirectRegistrants/AddSignatoryAndCompleteRequestHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace EA.Weee.RequestHandlers.Organisations.DirectRegistrants
+{
+    using DataAccess;
+    using EA.Prsd.Core.Mediator;
+    using EA.Weee.DataAccess.DataAccess;
+    using EA.Weee.Domain;
+    using EA.Weee.Domain.Producer;
+    using EA.Weee.Requests.Organisations.DirectRegistrant;
+    using Mappings;
+    using Security;
+    using System.Data.Entity;
+    using System.Threading.Tasks;
+
+    internal class AddSignatoryAndCompleteRequestHandler : SubmissionRequestHandlerBase, IRequestHandler<AddSignatoryAndCompleteRequest, bool>
+    {
+        private readonly WeeeContext weeeContext;
+
+        public AddSignatoryAndCompleteRequestHandler(IWeeeAuthorization authorization,
```

Comment:
@stevelee-civica  is there unit tests for this one...

Reply:
yes, AddSignatoryAndCompleteRequestHandlerTests